librz/debug/p/debug_dmp.c

```diff
@@ -136,7 +136,8 @@ static bool rz_debug_dmp_init(RzDebug *dbg, void **user) {
 		const char *symstore = dbg->corebind.cfgGet(dbg->corebind.core, "pdb.symstore");
 		char *pdbpath, *exepath;
 		if (winkd_download_module_and_pdb(&mod, server, symstore, &exepath, &pdbpath)) {
-			dbg->corebind.cmdf(dbg->corebind.core, "idp %s", pdbpath);
+			// TODO: Convert to API call
+			dbg->corebind.cmdf(dbg->corebind.core, "idp \"%s\"", pdbpath);
```

Comment:
Just use `rz_core_bin_pdb_load` , should work.

Reply:
@ret2libc it's a function from `librz/core/` but this is the `librz/debug/`

Reply:
oh sorry my bad...